README.md

```diff
@@ -1,411 +1,354 @@
-      OpenSSL::HMAC.digest('sha1', @secret_key, policy)
-    )
-  end
-
-  def policy
-    Base64.strict_encode64({
-      expiration: @options[:expiration].utc.iso8601,
-      conditions: conditions
-    }.to_json)
-  end
-
-  def to_json
-    {
-      url: "https://#{@options[:bucket]}.s3.amazonaws.com",
-      credentials: {
-        AWSAccessKeyId: @access_key,
-        policy:         policy,
-        signature:      signature,
-        acl:            @options[:acl],
-        key:            @options[:key]
-      }
-    }.to_json
-  end
-
-  private
-
-  def conditions
-    dynamic_key = @options[:key].include?('${filename}')
-    prefix = @options[:key][0..(@options[:key].index('${filename}') - 1)]
-
-    conditions = (@options[:conditions] || []).map(&:clone)
-    conditions << { bucket: @options[:bucket] }
-    conditions << { acl: @options[:acl] }
-    conditions << { key: @options[:key] } unless dynamic_key
-    conditions << ['starts-with', '$key', prefix] if dynamic_key
-    conditions << ['starts-with', '$Content-Type', '']
-  end
-
-  private
-
-  def require_options(options, *keys)
-    missing_keys = keys.select { |key| !options.key?(key) }
-    return unless missing_keys.any?
-    raise ArgumentError, missing_keys.map { |key| ":#{key} is required to generate a S3 upload policy." }.join('\n')
-  end
-end
-```
-
-After setting up your S3 bucket and server, you can start writing the code to upload files directly to S3!
-
-```javascript
-import Ember from 'ember';
-
-const RSVP = Ember.RSVP;
-const set = Ember.set;
-
-export default Ember.Route.extend({
-  actions: {
-    uploadImage: function (file) {
-      let model = this.modelFor(this.routeName);
-      RSVP.cast(Ember.$.get('/api/s3_direct')).then(function (response) {
-        return file.upload(response.url, {
-          data: response.credentials
-        });
-      }).then(function (response) {
-        set(model, 'url', response.headers.Location);
-        return model.save();
-      });
-    }
-  }
-});
-```
-
-
-## Contributing
-
-Contributors are welcome! Please provide a reproducible test case. Details will be worked out on a case-per-case basis. Maintainers will get in touch when they can, so delays are possible. For contribution guidelines, see [Contributing](CONTRIBUTING.md) and [code of conduct](CONDUCT.md).
-
-
-## Publishing
-
-* `ember github-pages:commit --message "Releasing docs"`
-
-For more information on using ember-cli, visit [https://ember-cli.com/](https://ember-cli.com/).
+# {{file-upload}} [![Build Status](https://travis-ci.org/adopted-ember-addons/ember-file-upload.svg)](https://travis-ci.org/adopted-ember-addons/ember-file-upload) [![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-file-upload/badges/gpa.svg)](https://codeclimate.com/github/adopted-ember-addons/ember-file-upload) [![Ember Observer Score](https://emberobserver.com/badges/ember-file-upload.svg)](https://emberobserver.com/addons/ember-file-upload)
```

Comment:
Not sure if this is how it's done. But if we add the `branch=latest` queryParam to the Travis Badge URL we get a passing build badge.

This seems consistent with [the Travis CI page](https://travis-ci.org/adopted-ember-addons/ember-file-upload).

```suggestion
# {{file-upload}} [![Build Status](https://travis-ci.org/adopted-ember-addons/ember-file-upload.svg)](https://travis-ci.org/adopted-ember-addons/ember-file-upload) [![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-file-upload/badges/gpa.svg)](https://codeclimate.com/github/adopted-ember-addons/ember-file-upload) [![Ember Observer Score](https://api.travis-ci.org/adopted-ember-addons/ember-file-upload.svg?branch=latest)](https://emberobserver.com/addons/ember-file-upload)
```

Reply:
Could catch! But in suggested change you have changed the Ember Observer URL and not the one for TravisCI. Maybe also a good one for another PR as the changes to README are already very hard to review due to the conversion from `CRLF` to `LF` line endings.

Reply:
Oops. I corrected my suggestion anyway. Will make a new PR.

Reply:
In #277